conf/application.conf

```diff
@@ -58,9 +60,13 @@ mongodb {
   encryption.key = "QmFyMTIzNDVCYXIxMjM0NQ=="
   uri = "mongodb://localhost:27017/"${appName}
   timeToLive = 30days
-  replaceIndexes = true
+  journeyAnswersTimeToLive = 30days
+  replaceIndexes = false
+  replaceJourneyAnswersIndexes = true
 }
 
+crypto.key = "QmFyMTIzNDVCYXIxMjM0NQ=="
```

Comment:
We can probably reuse the other encryption key and TTL as they have the same values